Avoid stripping the toolchain to prevent failing a QA check in Yocto.

Upstream-Status: Inappropriate [oe-specific qa check]
diff --git a/cmake/wasi-sdk-toolchain.cmake b/cmake/wasi-sdk-toolchain.cmake
index 0e1cf4e..7c58d6d 100644
--- a/cmake/wasi-sdk-toolchain.cmake
+++ b/cmake/wasi-sdk-toolchain.cmake
@@ -91,7 +91,7 @@ ExternalProject_Add(llvm-build
     -DLLVM_ENABLE_LIBXML2=OFF
     # Pass `-s` to strip symbols by default and shrink the size of the
     # distribution
-    -DCMAKE_EXE_LINKER_FLAGS=-s
+#    -DCMAKE_EXE_LINKER_FLAGS=-s
     ${llvm_cmake_flags_list}
   # See https://www.scivision.dev/cmake-externalproject-list-arguments/ for
   # why this is in `CMAKE_CACHE_ARGS` instead of above
